packages/nocodb-sdk/src/lib/is.ts

```
@@ -0,0 +1,207 @@
+ * Checks if a value is a function.
+ *
+ * @param value - The value to check.
+ * @returns {boolean} - True if the value is a function, false otherwise.
+ *
+ * @example
+ * ```typescript
+ * const value = () => {};
+ * console.log(ncIsFunction(value)); // true
+ * ```
+ */
+export function ncIsFunction(value: any): value is Function {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)